AdlsSDKGettingStarted/Program.cs

```diff
@@ -56,55 +69,50 @@ public static void Main(string[] args)
                 }
 
                 // Get the properties of the file
-                var directoryEntry = client.GetDirectoryEntry(fileName);
-                PrintDirectoryEntry(directoryEntry);
+                PathProperties pathProperties = file.GetProperties();
+                PrintDirectoryEntry(pathProperties);
 
                 // Rename a file
                 string destFilePath = "/Test/testRenameDest3.txt";
-                client.Rename(fileName, destFilePath, true);
+                file.Rename(destFilePath);
```

Comment:
We should reassign the `file` variable to the client returned and print out the new URI so people realize that the old `file` is now pointing nowhere.

Reply:
@tg-msft Updated.

Reply:
I don't see an update?  After you call `Rename`, `file` is pointing at a bogus URI that no longer exists.  We return a new file client in the response which we should pull out and reassign to `file`.